apps/datafeeder/src/app/presentation/components/map-view/map-view.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+
+import { MapViewComponent } from './map-view.component'
+
+describe('MapViewComponent', () => {
+  let component: MapViewComponent
+  let fixture: ComponentFixture<MapViewComponent>
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [MapViewComponent],
+    }).compileComponents()
+  }))
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(MapViewComponent)
+    component = fixture.componentInstance
+    fixture.detectChanges()
+  })
+
+  it('should create', () => {
```

Comment:
I expect this component to have more tests.
At least check that the title is well displayed for the moment.